libfwupd/fwupd-remote.c

```diff
@@ -978,6 +978,14 @@ fwupd_remote_load_from_filename(FwupdRemote *self,
 			fwupd_remote_remove_flag(self,
 						 FWUPD_REMOTE_FLAG_AUTOMATIC_SECURITY_REPORTS);
 	}
+
+	/* old versions of fwupd used empty strings to mean "unset" and the package manager
+	 * might not have replaced the file marked as as a config file due to modification */
```

Comment:
sorry for the nitpick. logic looks good
```suggestion
	 * might not have replaced the file marked as a config file due to modification */
```